components/channel_header/channel_header.tsx

```diff
@@ -492,29 +576,43 @@ class ChannelHeader extends React.PureComponent {
                     >
                         <Markdown
                             message={headerText.replace(/\n+/g, ' ')}
-                            options={this.getHeaderMarkdownOptions(channelNamesMap)}
-                        /></div>
+                            options={this.getHeaderMarkdownOptions(
+                                channelNamesMap,
+                            )}
+                        />
+                    </div>
                     <span
                         className='header-description__text'
                         onClick={this.handleFormattedTextClick}
-                        onMouseOver={() => this.showChannelHeaderPopover(headerText)}
-                        onMouseOut={() => this.setState({showChannelHeaderPopover: false})}
+                        onMouseOver={() =>
+                            this.showChannelHeaderPopover(headerText)
+                        }
+                        onMouseOut={() =>
+                            this.setState({showChannelHeaderPopover: false})
+                        }
                         ref={this.headerDescriptionRef}
                     >
-
                         <Overlay
                             show={this.state.showChannelHeaderPopover}
                             placement='bottom'
                             rootClose={true}
-                            target={this.headerDescriptionRef.current}
+                            target={this.headerDescriptionRef.current as HTMLSpanElement}
```

Comment:
this is being typed out correctly at the top, so there is no need to cast this again here